tee-worker/litentry/primitives/src/lib.rs

```diff
@@ -107,15 +104,9 @@ pub enum LitentryMultiSignature {
 	/// An ECDSA/keccak256 signature. An Ethereum signature. hash message with keccak256
 	#[codec(index = 3)]
 	Ethereum(EthereumSignature),
-	/// Same as above, but the payload bytes are prepended with a readable prefix and `0x`
-	#[codec(index = 4)]
-	EthereumPrettified(EthereumSignature),
 	/// Bitcoin signed message, a hex-encoded string of original &[u8] message, without `0x` prefix
-	#[codec(index = 5)]
+	#[codec(index = 4)]
```

Comment:
Is this a breaking change ?

Reply:
Yes, it is, although I already managed it on IDH side. It shouldn't impact current ts-test as no prettified versions are used.

I'll add the breaking-change tag